content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -143,27 +157,50 @@ is recommended to use
 [taints on nodes](/docs/concepts/configuration/taint-and-toleration/)
 to schedule those pods onto the right nodes.
 
-## PodSecurityPolicy Annotations
+## PodSecurityPolicy
+
+To control which sysctls can be set in pods, specify the
+`forbiddenSysctls` and/or `allowedUnsafeSysctls` fields in the PodSecurityPolicy.
+
+By default, all safe sysctls in the whitelist are allowed.
+
+Both `forbiddenSysctls` and `allowedUnsafeSysctls` are lists of plain sysctl names
+or sysctl patterns (which end with `*`). The string `*` matches all sysctls.
 
-The use of sysctl in pods can be controlled via annotation on the PodSecurityPolicy.
+The `forbiddenSysctls` field excludes specific sysctls, and can include a
+combination of safe and unsafe ones. To forbid setting any sysctls, use `*` on
+its own.
 
-Sysctl annotation represents a whitelist of allowed safe and unsafe sysctls
-in a pod spec. It's a comma-separated list of plain sysctl names or sysctl patterns
-(which end in `*`). The string `*` matches all sysctls.
+If you specify any unsafe sysctl in the `allowedUnsafeSysctls` field and it is
+not present in the `forbiddenSysctls` field, that sysctl can be used in Pods under
+this PodSecurityPolicy. In order to allow all unsafe sysctls in the PodSecurityPolicy
+to be set (except for those explicitly forbidden by `forbiddenSysctls`),
+use `*` on its own.
```

Comment:
unfortunately this does not work: you cannot use `*` and forbid some in `forbiddenSysctls` at the same time. Just drop the parenthesis.